lib/utils.py

```diff
@@ -23,7 +23,8 @@
 
 arch = machine()
 
-HTTP_OK = xrange(200, 299)
+# 300 level HTTP responses are also ok, such as redirects, which many sites have and load
+HTTP_OK = xrange(200, 399)
```

Comment:
Should we throw [418](http://httpstat.us/418) in there too? ;)

Reply:
lol, no tea for me, only coffee